### PR TITLE
 Multiple edge routers support for Contrail

### PR DESCRIPTION
**Pull request by vpramo**
Testing the Multiple edge routers support in Jiocloud Contrail

### CODE REVIEW

**Comment by vpramo** [comment]
please test this

**Comment by vpramo** [comment]
please test this

**Comment by vpramo** [comment]
please test this

**Comment by vpramo** [comment]
please test this

**Comment by vpramo** [comment]
please test this

**Comment by vpramo** [comment]
please test this

**Comment by vpramo** [comment]
please test this

**Comment by vpramo** [comment]
please test this

**Comment by vpramo** [comment]
@hkumarmk  and @sorenh  Please review , the build is successful.

**Comment by hkumarmk** [comment]
I see it is working fine - added two routers with provided IP address and name. 

$ curl http://localhost:8082/bgp-routers
{"bgp-routers": [{"href": "http://localhost:8082/bgp-router/5745a0c5-5888-45a1-9b82-0240f2341d25", "fq_name": ["default-domain", "default-project", "ip-fabric", "__default__", "ct1-testjenkins-puppet-rjil-gate-1651"], "uuid": "5745a0c5-5888-45a1-9b82-0240f2341d25"}, {"href": "http://localhost:8082/bgp-router/ae82ed7f-89bb-4025-846c-c4d4fd25353b", "fq_name": ["default-domain", "default-project", "ip-fabric", "__default__", "router1"], "uuid": "ae82ed7f-89bb-4025-846c-c4d4fd25353b"}, {"href": "http://localhost:8082/bgp-router/8cd35124-5d80-4b6c-840c-b7746480c89e", "fq_name": ["default-domain", "default-project", "ip-fabric", "__default__", "router2"], "uuid": "8cd35124-5d80-4b6c-840c-b7746480c89e"}]}

File: .fixtures.yml [hunk 1]
```diff
@@ -32,7 +32,8 @@ fixtures:
       repo: https://github.com/saz/puppet-ssh
       ref: v2.3.6
     contrail:
-      repo: https://github.com/jiocloud/jiocloud-contrail
+      repo: https://github.com/vpramo/jiocloud-contrail
```
Comment (hkumarmk) [inline .fixtures.yml]: This is merged, please change the reference, so it is ready to merge.

**Comment by vpramo** [comment]
please test this

File: hiera/data/env/at.yaml [hunk 2]
```diff
@@ -37,3 +37,8 @@ rjil::neutron::contrail::fip_pools:
     tenants: ['tempest','mgmt_tenant','internal_tenant']
 
 contrail::vrouter::vgw_subnets: ['100.1.0.0/24']
+
+contrail::edge_routers:
+  router1: {host_address: '1.1.1.1'}
+  router2: {host_address: '1.1.1.2'}
+
```
Comment (vpramo) [inline hiera/data/env/at.yaml]: @hkumarmk is this required , these are dummy values

Comment (hkumarmk) [inline hiera/data/env/at.yaml]: yes it is, because that is the way you can test this capability in at/gate

Comment (vpramo) [inline hiera/data/env/at.yaml]: @hkumarmk ok cool, please merge  the checks have passed and i verified the router info in bgp routers.

**Comment by vpramo** [comment]
please test this

**Pull request merged**